src/styles/blocks/stars-container.scss

```
@@ -0,0 +1,21 @@
+.stars-container {
```

I'd recommend to name just .stars
```suggestion
.stars {
```